hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This patch fixes file field handling for django forms when using :func:`~hypothesis.extra.django._impl.from_form` as reported in (:issue:`4093`).
+
+Thanks to @thesage21 for this fix!
```

Comment:
```suggestion
This patch fixes file field handling for django forms when using
:func:`~hypothesis.extra.django.from_form` (:issue:`4093`).

Thanks to Arjoonn Sharma for this fix!
```

Referring to the public location of the function, plus a slight style tweak to match other changelog entries - if you prefer to be thanked by handle that's also fine, of course, but by default I'd match your entry in the authors list.